man/fi_domain.3.md

```diff
@@ -348,12 +352,11 @@ are defined.
   received operations.
 
 *FI_PROGRESS_CONTROL_UNIFIED*
-: This progress model indicates that the user will synchronize progressing the
-  data and control operations themselves (i.e. this allows the control interface
-  to NOT be thread safe). It is only valid for control progress (not data progress).
-  Setting control=FI_PROGRESS_CONTROL_UNIFIED, data=FI_PROGRESS_MANUAL, and
-  threading=FI_THREAD_DOMAIN/FI_THREAD_COMPLETION allows Libfabric to remove all
-  locking in the critical data progress path.
+: This progress model indicates that the user will synchronize progressing
```

Comment:
If we remove control progress, aren't we assuming data and ctrl progress are unified?

Reply:
That is a question I would like to discuss. It is possible that both use the same model (auto or manual) but progress independently. But does it make practical sense to keep them separated? Or are there technical difficulties for all providers to support unified progress?

Reply:
cc @shefty

Reply:
My intent was to remove the arbitrary separation between data and control progress.  There's only progress, which can be manual or auto.  Heck, even trying to determine if an async operation that generates a completion meets the requirement for being manual versus auto is non-trivial.

The only "non-data" asynchronous operation that is left is connection establishment.  Such operations are reported to an EQ, which already provides separation from data transfer completions.  Connections require user intervention in order to complete.  I don't know what auto progress really means there.